deps/build.jl

```diff
@@ -2,7 +2,13 @@
 
 path = joinpath(dirname(@__FILE__), "..", "src")
 
-if is_windows()
+version = "0.1.0"
+
+if is_apple()
+  download("https://www.dropbox.com/s/syt6q6qv6ehc4p0/nnlib-$version.dylib?dl=1",
```

Comment:
how about hosting the binary via GitHub release page?

Reply:
I'm working on getting a real host for these, for now these should hold up.